index.html

```diff
@@ -32,7 +32,7 @@
     <!-- nav start -->
     <nav>
       <div class="logo">
-        <img src="IMG/my-logo.png" alt="">
+        <img src="IMG/my-logo.png" alt="" />
```

Comment:
- [ ] [Optional] Wherever you have images, it's not good to leave `alt=' '` as empty, you should give them explicit descriptive value, for the sake of you site's accessibility!